src/components/Troubleshoot/TroubleShootCard.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { withStyles } from '@material-ui/core/styles';
+import { useSelector } from 'react-redux';
+import { RootState } from '../../store/reducers';
+import { styles } from './TroubleShootStyles';
+import Button from '@material-ui/core/Button';
+import HelpOutlineIcon from '@material-ui/icons/HelpOutline';
+type TroubleShootProps = {
+  helpId: string,
+  classes: any,
+}
+
+const TroubleShootCard = ({ helpId, classes }: TroubleShootProps) => {
+
+  const troubleshoot = useSelector((state: RootState) => state.troubleshoot);
+  const help = troubleshoot[helpId];
+  const createOptions = (options) => {
+    return (
+      <div>
+        {Object.keys(options).map((option) => {
+          if (option == "error_code") {
+          } else {
+            let buttonType = `${classes.button}`;
+            if (option == "yes") {
+              buttonType = `${classes.button} ${classes.yes}`
+            } else if (option == "no") {
+              buttonType = `${classes.button} ${classes.no}`
+            }
+            return <Link to={'/Troubleshoot/' + options[option]} style={{ textDecoration: 'none' }}>
```

Comment:
Is it possible to consolidate styles into the styles file for all of these inline styles?

Reply:
i'll take care of styles in the next pr